codalab/lib/worksheet_util.py

```diff
@@ -571,8 +572,9 @@ def flush_bundles():
         elif mode == 'contents' or mode == 'image' or mode == 'html':
             def raiseUsageError():
                 raise UsageError('Expected \'% display ' + mode + ' (genpath)\', but got \'% display ' + ' '.join([mode] + args) + '\'')
-            for bundle_info in bundle_infos:
+            for i, bundle_info in bundle_infos:
```

Comment:
If this `i` has some meaning beside the number of iterations so far in the for loop, might be helpful to give it a more explicit name like `item_index`